src/docs/cmdstan-guide/getting-started.tex

```diff
@@ -15,7 +15,11 @@ \section{Installation}
   download.
 \item The \code{make} utility program. \\
   This is not strictly necessary, but will make the build process
-  easy. The rest of the documentation assumes \code{make} is available.
+  easy. On the Windows platform the \code{mingw32-make} variant is
```

Comment:
Just say "On Windows \code{mingw32-make} is needed to build \CmdStan. This is a \code{make} variant and can be installed as part of Rtools ...".